src/logger.ts

```diff
@@ -74,11 +77,15 @@ export class Logger {
 
   _log = (level: string, message: string, args: { [key: string]: any } = {}) => {
     const logEvent: LogEvent = { level, message, _time: new Date(Date.now()).toISOString(), fields: {} };
+    // check if there is args attached to the logger
+    if (Object.keys(this.args).length) {
+      logEvent.fields = { ...this.args };
+    }
```

Comment:
Is it possible to pass `this.args || {}` in the `fields: {}` initialisation above?

Reply:
you are right, that's better. updated it.